VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/dao/LoginDao.java

```diff
@@ -0,0 +1,38 @@
+package edu.vt.cs.vtcare.vtcareservice.dao;
+import edu.vt.cs.vtcare.vtcareservice.db.VTCareJDBC;
+
+import java.sql.*;
+
+public class LoginDao {
+    private final Connection connection;
+
+    private static final String LOGIN =
+            "SELECT id FROM %s WHERE email = ? AND password = ?";
+
+    public LoginDao() throws Exception {
+        connection = VTCareJDBC.getInstance().getConnection();
+    }
+
+    public int validateLogin(String email, String password,
+                              boolean isProviderLogin) throws SQLException {
+
+        String finalString = String.format(LOGIN, (isProviderLogin) ?
+                "providers" : "patients");
+
+        try (PreparedStatement statement = connection.prepareStatement(finalString)) {
+
+            statement.setString(1, email);
+            statement.setString(2, password);
+            ResultSet res = statement.executeQuery();
+
+            if(res.next()){
+                return res.getInt(1);
```

Comment:
Add a comment to describe as to why 1st column is being sent in the return value. It is not readable as it is.

Reply:
Again, it is clear to you as the developer and to me as your team mate because we are working on this project for last 3 months. 
Also, I in addition to commenting, please remove the column number completely from the statement and use column name instead. using 1, 2 etc is very unreadable and makes your code difficult to understand. 

getInt("id") is much more readable than getInt(1).

Reply:
Again, since we only return 1 thing, adding "id" would still mean the exact same thing.

Reply:
I am not talking about the semantics here. It is bad from a readability point of few. Someone else reading your code would have to look in the database table to see which what value is being returned. And that's inimical to readability. Please change it to the column name.

Reply:
These are good practices and not debatable. More, so when you are pretty particular about readability.